projects/core/components/primitive-textfield/primitive-textfield.component.ts

```diff
@@ -111,6 +115,9 @@ export class TuiPrimitiveTextfieldComponent
     @Output()
     readonly valueChange = new EventEmitter<string>();
 
+    /**
+     * @deprecated TODO: remove
```

Comment:
need information on what will be replaced and in what version

Reply:
In many years this was never needed, I believe we can just remove it in 3.0. I'll add the version.